tests/test_processors.py

```diff
@@ -491,25 +500,21 @@ def test_gets_exc_info_on_bool(self):
         try:
             raise ValueError("test")
         except ValueError:
-            d = format_exc_info(None, None, {"exc_info": True})
+            d = ExceptionFormatter()(None, None, {"exc_info": True})
 
         assert "exc_info" not in d
         assert 'raise ValueError("test")' in d["exception"]
         assert "ValueError: test" in d["exception"]
 
-    def test_exception_on_py3(self, monkeypatch):
+    def test_exception_on_py3(self):
```

Comment:
While you're touching it :)

```suggestion
    def test_exception(self):
```